packages/dev/docs/src/docs.css

```diff
@@ -90,17 +90,17 @@ article {
 }
 
 article h1 {
-  margin-top: 0 !important;
-  font-size: 58px !important;
```

Comment:
These are needed to override spectrum for the top heading of each page and the first paragraph with the description of the component. After removing the importants they are smaller now.

Maybe make the selector more specific? Something like `article > h1`? That way h1s inside sub elements aren't affected?

Reply:
I created the PR prematurelly. I kept working on it and found something specific enough. I found `article > h1` wasn't specific enough.